op-challenger/config/config.go

```diff
@@ -72,7 +73,14 @@ func ValidTraceType(value TraceType) bool {
 	return false
 }
 
-const DefaultCannonSnapshotFreq = uint(1_000_000_000)
+const (
+	DefaultCannonSnapshotFreq = uint(1_000_000_000)
+	// DefaultGameWindow is the default maximum time duration in the past
+	// that the challenger will look for games to progress.
+	// The default value is 10.5 days, which is a 3.5 day resolution buffer
+	// plus the 7 day game finalization window.
+	DefaultGameWindow = time.Duration(int64(907_200))
```

Comment:
```suggestion
	DefaultGameWindow = time.Duration(252 * time.Hour)
```

Side note: if we did a full number of days, we could do `11 * 24 * time.Hour`. Golang doesn't provide a duration greater than 1 hour. (Scroll to the bottom of this section: https://pkg.go.dev/time#pkg-constants)

Reply:
Changed to 11 to keep the round number. For now the extra 0.5 days in the naive approach most likely won't make a difference and just leans toward being conservative in approach.